Statut.md

```diff
@@ -0,0 +1,169 @@
+2. Partnerem Fundacji może być osoba fizyczna lub prawna, stanowiąca podmiot, który:
+    a) realizuje zadania społecznie użyteczne lub zamierza rozpocząć ich realizację, poprzez rozpoczęcie współpracy z Fundacją (partner);
+    b) wspomaga Fundację w realizacji jej celu w wymiarze materialnym lub niematerialnym (patron).
+3. Fundacja tworzy tylko i wyłącznie otwarcie dostępne i nieodpłatne oprogramowanie, publikowane i dystrybuowane za pośrednictwem platformy internetowej GitHub.com, udostępniane na licencji MIT lub CC BY-SA 4.0 lub podobnej, pozwalającej na nieograniczone kopiowanie, dystrybucję oraz wykorzystanie zarówno w celach prywatnych, jak i komercyjnych.
+4. Fundacja prowadzi regularne spotkania edukacyjne w formie wspólnoty praktyki, służących wymianie doświadczeń i organizowaniu współpracy, podczas których bardziej doświadczeni uczą i pomagają w programowaniu mniej doświadczonym wolontariuszom, wspólnie rozwijając oprogramowanie, będące przedmiotem działalności Fundacji (projekty społecznie użyteczne) lub wspomagającej jej funkcjonowanie (projekty własne).
+5. Wolontariuszami Fundacji mogą być osoby o różnych kompetencjach w tym: programiści, testerzy oprogramowania, specjaliści w zakresie UX, menedżerowie projektów, aktywiści społeczni, pracownicy organizacji non-profit, badacze społeczni, prawnicy i inni, których wiedza oraz umiejętności mogą przyczynić się do realizacji celu Fundacji.
+6. Fundacja realizuje swoje cele poprzez prowadzenie odpłatnej i nieodpłatnej działalności pożytku publicznego w zakresie wskazanym w §8 oraz art. 4 ust. 1 pkt. 11-13 w Ustawie o działalności pożytku publicznego i wolontariacie z dnia 24 kwietnia 2003 wraz z późniejszymi zmianami.
+7. Fundacja może wspierać wszelkie działania, które są spójne z jej misją, wizją oraz służą budowaniu lepszego społeczeństwa.
+
+§10 Działania wspierające
+Fundacja prowadzi również działania dodatkowe, wspierające realizację celów w tym:
+	1) gromadzenie i przetwarzanie danych dla realizacji celów Fundacji;
+	2) organizację szkoleń, warsztatów, kursów, wykładów, spotkań, doradztwa specjalistycznego oraz publikacji materiałów związanych z celami Fundacji;
+	3) organizowanie spotkań, służących wymianie doświadczeń osób i podmiotów prowadzących działalność w zakresie zbieżnym z celami Fundacji;
+	4) budowanie społeczności, skupiającej różnorodnych ludzi zainteresowanych działalnością społecznie użyteczną i celami Fundacji;
+	5) prowadzenie lub wspieranie badań naukowych i rozwojowych związanych z celami Fundacji oraz rozpowszechnienie ich wyników;
+	6) publikacja wyników badań i opracowań, podsumowań gromadzonych danych oraz materiałów szkoleniowo-dydaktycznych, o tematyce związanej z celami Fundacji;
+	7) Organizowanie i udział w akcjach oraz kampaniach społecznych, seminariach, pokazach, imprezach edukacyjnych i kulturalnych;
```

Comment:
od tego punktu zaczynają się akapity rozpoczynające się wielką literą, proponuję zmienić je na małe, bo w końcu to jedno długie zdanie